yt_dlp/extractor/maariv.py

```diff
@@ -0,0 +1,70 @@
+from .common import InfoExtractor
+import re
```

Comment:
And finally we need to import all the used helper functions
```suggestion
from .common import InfoExtractor
from ..utils import (
    int_or_none,
    parse_qs,
    parse_resolution,
    unified_timestamp,
    url_or_none,
)
from ..utils.traversal import traverse_obj
```